server/svix-server/Cargo.toml

```diff
@@ -75,6 +75,8 @@ strum_macros = "0.24"
 strum = { version = "0.24", features = ["derive"] }
 form_urlencoded = "1.1.0"
 lapin = "2.1.1"
+sentry = { version = "0.31.5", features = ["tracing"] }
+sentry-tracing = "0.31.5"
```

Comment:
The tracing crate is re-exported, looks like.
https://github.com/getsentry/sentry-rust/blob/master/sentry/src/lib.rs#L221-L224

Should be available as `sentry::integrations::tracing` when the feature is enabled.